test/e2e/extra_test.go

```diff
@@ -63,5 +63,19 @@ var _ = Describe("Deploy cluster", Label("cluster-extra-ns"), func() {
 				},
 			),
 		),
+		Entry("Advanced cluster configuration with no backup and no user", Label("ns-autoscaling"),
+			model.NewTestDataProvider(
+				"operator-ns-autoscaling",
+				model.AProject{},
+				model.NewEmptyAtlasKeyType().UseDefaulFullAccess(),
+				[]string{"data/atlascluster_with_autoscaling.yaml"},
+				[]string{},
+				[]model.DBUser{},
```

Comment:
Do we need this user?

Reply:
[Seems like](https://github.com/commercetools/mongodb-atlas-kubernetes/blob/8802b364bc593694b8c94fde51119a2c6d50d07f/test/e2e/model/dataprovider.go#L13)

Reply:
I answered this question here:
https://github.com/commercetools/mongodb-atlas-kubernetes/pull/1#discussion_r911784662